### PR TITLE
Announce AWS marketplace offering

### PR DESCRIPTION
**Pull request by iskerrett**
AWS Marketplace announcement.

### CODE REVIEW

**Review by MarianRaphael — APPROVED** [review]
In addition, I would add the link to our AWS offering:
https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa

File: src/blog/2023/08/aws-marketplace-announce.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+title: FlowFuse is now available on AWS Marketplace
+subtitle: Making is easier to run Node-RED and FlowFuse on AWS Cloud
+description: This is an announcement that customers can install FlowFuse on AWS Cloud from the AWS Marketplace.
+date: 2023-08-18
+authors: ["ian-skerrett"]
+image: blog/2023/08/images/UNS/UNS-Headline.svg
+tags:
+    - posts
+    - news
+---
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/aws-marketplace-announce.md:11]:
```suggestion
---

```

**Comment by hardillb** [comment]
Will want to include a link to the docs as well which will be here 

https://flowfuse.com/docs/install/docker/aws-marketpace/ 

when https://github.com/flowforge/flowforge/pull/2618 is merged

**Comment by iskerrett** [comment]
> In addition, I would add the link to our AWS offering: https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa

Yes of course. :-)

**Comment by ZJvandeWeg** [comment]
@MarianRaphael Can you create a doc update on the main repo with a backport next week?

File: src/blog/2023/08/aws-marketplace-announce.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+title: FlowFuse is now available on AWS Marketplace
+subtitle: Making is easier to run Node-RED and FlowFuse on AWS Cloud
+description: This is an announcement that customers can install FlowFuse on AWS Cloud from the AWS Marketplace.
+date: 2023-08-18
+authors: ["ian-skerrett"]
+image: blog/2023/08/images/UNS/UNS-Headline.svg
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/aws-marketplace-announce.md:7]: Consider an image from unsplash if you want to publish today: https://unsplash.com/photos/M2524ncJQ40

File: src/blog/2023/08/aws-marketplace-announce.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+FlowFuse on the AWS Marketplace is available under the Apache License c2.0 open source license. Customers can use FlowFuse free of charge but will need to pay AWS EC2 usage for hosting FlowFuse. FlowFuse offers commercially licensed Premimum and Enterprise tiers that includes enterprise oriented features, including:
+- 24/7 technical support
+- Single Sign-on
+- High Availability for Node-RED applications
+- Remote device management
+
+Customers can easily upgrade to the premium or enterprise tier by obtaining a commercial license from FlowFuse.
+
+FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](https://flowfuse.com/docs/install/docker/aws-marketpace/) and give it a try and let us know what you think.
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/aws-marketplace-announce.md:29]:
```suggestion
FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](/docs/install/docker/aws-marketpace/) and give it a try and let us know what you think.
```

File: src/blog/2023/08/aws-marketplace-announce.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+description: This is an announcement that customers can install FlowFuse on AWS Cloud from the AWS Marketplace.
+date: 2023-08-18
+authors: ["ian-skerrett"]
+image: blog/2023/08/images/UNS/UNS-Headline.svg
+tags:
+    - posts
+    - news
+---
+
+Today, we are excited to announce that FlowFuse is now available from the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). This makes it very easy for customers to install and run Node-RED and FlowFuse on the AWS Cloud.
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/aws-marketplace-announce.md:13]: Nit, but I like this a bit better

```suggestion
Many customers want to run FlowFuse in their own cloud environment, AWS being a great example. Today we're excited to announce that FlowFuse is now available from the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). This makes it very easy for customers to install and run Node-RED and FlowFuse within minutes.

```

File: src/blog/2023/08/aws-marketplace-announce.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- DevOps deliver pipelines that support a software development lifecycle for Node-RED development. Pipelines can be setup to establish development, test and production environments for Node-RED instances.
+- Snapshot of Node-RED instances to create a version history of changes to Node-RED applications. This also includes the ability to rollback to a previous version of a Node-RED instance.
+
+FlowFuse on the AWS Marketplace is available under the Apache License c2.0 open source license. Customers can use FlowFuse free of charge but will need to pay AWS EC2 usage for hosting FlowFuse. FlowFuse offers commercially licensed Premimum and Enterprise tiers that includes enterprise oriented features, including:
+- 24/7 technical support
+- Single Sign-on
+- High Availability for Node-RED applications
+- Remote device management
+
+Customers can easily upgrade to the premium or enterprise tier by obtaining a commercial license from FlowFuse.
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/aws-marketplace-announce.md:27]:
```suggestion
Customers can easily upgrade to the premium or enterprise tier by obtaining a commercial license from FlowFuse. To understand what FlowFuse can do for your use-case, please [book a demo](/book-demo).
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Review by ZJvandeWeg — APPROVED** [review]

File: src/blog/2023/08/aws-marketplace-announce.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+---
+title: FlowFuse is now available on AWS Marketplace
+subtitle: Making is easier to run Node-RED and FlowFuse on AWS Cloud
+description: This is an announcement that customers can install FlowFuse on AWS Cloud from the AWS Marketplace.
+date: 2023-08-18
+authors: ["ian-skerrett"]
+image: blog/2023/08/images/flowfuse-aws.png
+tags:
+    - posts
+    - news
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/aws-marketplace-announce.md:10]:
```suggestion
    - news
    - flowfuse
```

File: src/blog/2023/08/aws-marketplace-announce.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+---
+title: FlowFuse is now available on AWS Marketplace
+subtitle: Making is easier to run Node-RED and FlowFuse on AWS Cloud
+description: This is an announcement that customers can install FlowFuse on AWS Cloud from the AWS Marketplace.
+date: 2023-08-18
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/aws-marketplace-announce.md:5]:
```suggestion
date: 2023-08-21
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Comment by hardillb** [comment]
Docs will go live with the next Website publish

https://github.com/flowforge/flowforge/pull/2622

**Comment by ZJvandeWeg** [comment]
I've rebased this PR and hopefully that fixes the doc link being broken

File: src/blog/2023/08/aws-marketplace-announce.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+FlowFuse on the AWS Marketplace is available under the Apache License c2.0 open source license. Customers can use FlowFuse free of charge but will need to pay AWS EC2 usage for hosting FlowFuse. FlowFuse offers commercially licensed Premimum and Enterprise tiers that includes enterprise oriented features, including:
+- 24/7 technical support
+- Single Sign-on
+- High Availability for Node-RED applications
+- Remote device management
+
+Customers can easily upgrade to the premium or enterprise tier by obtaining a commercial license from FlowFuse. To understand what FlowFuse can do for your use-case, please [book a demo](/book-demo).
+
+FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](/docs/install/docker/aws-marketpace/) and give it a try and let us know what you think.
```
Comment (hardillb) [inline src/blog/2023/08/aws-marketplace-announce.md:31]:
```suggestion
FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](/docs/install/docker/aws-marketplace/) and give it a try and let us know what you think.
```

File: src/blog/2023/08/aws-marketplace-announce.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+FlowFuse on the AWS Marketplace is available under the Apache License c2.0 open source license. Customers can use FlowFuse free of charge but will need to pay AWS EC2 usage for hosting FlowFuse. FlowFuse offers commercially licensed Premimum and Enterprise tiers that includes enterprise oriented features, including:
+- 24/7 technical support
+- Single Sign-on
+- High Availability for Node-RED applications
+- Remote device management
+
+Customers can easily upgrade to the premium or enterprise tier by obtaining a commercial license from FlowFuse. To understand what FlowFuse can do for your use-case, please [book a demo](/book-demo).
+
+FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](/docs/install/docker/aws-marketplace/) and give it a try and let us know what you think.
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/aws-marketplace-announce.md:31]:
```suggestion
FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](/docs/install/docker/aws-marketplace/#installing-flowfuse-from-aws-market-place) and give it a try and let us know what you think.
```

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**